docs/sql-ref-datatypes.md

```diff
@@ -188,6 +192,8 @@ please use factory methods provided in
 |**DoubleType**|double or Double|DataTypes.DoubleType|
 |**DecimalType**|java.math.BigDecimal|DataTypes.createDecimalType()<br/>DataTypes.createDecimalType(*precision*, *scale*).|
 |**StringType**|String|DataTypes.StringType|
+|**CharType(length)**|String|Not supported|
+|**VarcharType(length)**|String|Not supported|
```

Comment:
shall we add the corresponding Java API?

Reply:
Addressed. Thank you